t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -197,9 +197,18 @@ sub _default_check_interval {
 
 sub wait_for_ajax {
     my $check_interval = _default_check_interval(shift);
-    while (!$_driver->execute_script("return jQuery.active == 0")) {
+    my $timeout        = 60 * 5;
+
+    while (!$_driver->execute_script('return window.jQuery && jQuery.active === 0')) {
+        if ($timeout <= 0) {
+            fail("Wait for jQuery");
+            return undef;
+        }
+
+        $timeout -= $check_interval;
         sleep $check_interval;
     }
+    pass("Wait for jQuery");
```

Comment:
I wonder if it wouldn't be more helpful if the pass message differs from the fail message in line https://github.com/os-autoinst/openQA/pull/2226/files#diff-77af89d1878c400551319858f891c558R204